### DISCUSSION TITLE
how to create randomized test cases

### DISCUSSION THREAD

**Discussion by Daniel-Chen-1002** [codewars/codewars.com#2424] [Q&A]
how to create randomized test cases in my kata?

**Comment by DonaldKellett** [comment]
View the test cases for Kata you have solved, and many of the more recent ones should include random tests, which can serve as a reference when adding random tests to your own Kata. There should also be a page on random tests in the [official docs](https://docs.codewars.com), but I cannot find them at the moment :sweat_smile: 

Also, as suggested in my response to your other query, you may wish to join our Discord chat and hop by the #help-author channel. Happy sparring!

EDIT: found the section on random tests: https://docs.codewars.com/authoring/guidelines/submission-tests#random-tests
